Formula/asymptote.rb

```diff
@@ -0,0 +1,39 @@
+class Asymptote < Formula
+  desc "Powerful descriptive vector graphics language"
+  homepage "https://asymptote.sourceforge.io"
+  url "https://downloads.sourceforge.net/project/asymptote/2.61/asymptote-2.61.src.tgz"
+  sha256 "f3ccaa6fde02a2d37fca771563597828530f94821e1c3fe3a5c7d81e7d398d3d"
+
+  depends_on "fftw"
+  depends_on "ghostscript"
+  depends_on "glm"
+  depends_on "gsl"
+  depends_on "readline"
```

Comment:
Can it run with macOS's libeedit?

Reply:
Apparently not... (I see no mention of histedit.h"

Reply:
I suggested editline support to upstream.  They have put in a fallback to editline if readline is not available (https://github.com/vectorgraphics/asymptote/commit/a0c44e0b438217ad04eb7833a8acc3a7bd571c0f).  This is in the git repo, and should become available with the next release.

How do you propose we proceed in the mean time?  I could add a comment about removing `depends_on "readline"` in the next release to make sure we remember to make the change...

Reply:
I say add a comment next to `readline` saying `remove in next release to use macOS editline`.

Reply:
done.